lib/core_types/bool.dart

```diff
@@ -14,6 +12,7 @@ ASTNode visitBoolProperties(ASTNode node, ASTNode left) {
       }
     default:
   }
+  return null;
```

Comment:
Do not return null. These changes are not also related to this pr.

Reply:
Added throw just like other types